include/grpcpp/impl/codegen/call_op_set.h

```diff
@@ -378,6 +385,7 @@ template <class M>
 Status CallOpSendMessage::SendMessage(const M& message, WriteOptions options) {
   write_options_ = options;
   serializer_ = [this](const void* message) {
+    send_message_called_ = true;
```

Comment:
Why is this assignment inside the lambda? Can't it be done before this point?

Reply:
Silly mistake - I meant to put it at the top of the method.